modules/token/token.service.ts

```diff
@@ -216,8 +216,8 @@ export class TokenService {
         return this.tokenPriceService.getHistoricalTokenPrices();
     }
 
-    public async purgeOldTokenPrices(): Promise<number> {
-        return this.tokenPriceService.purgeOldTokenPrices();
+    public async purgeOldTokenPricesForAllChains() {
+        await this.tokenPriceService.purgeOldTokenPricesForAllChains();
```

Comment:
async / await can be skipped here, because we don't use results within the function and  it returns a promise anyways